.travis.yml

```diff
@@ -24,5 +24,6 @@ before_script:
   - mysql -u root -e 'create database osem_test;'
   - RAILS_ENV=test bundle exec rake db:migrate --trace
 script:
+  - 'bundle exec rake factory_girl:lint'
```

Comment:
I have changed this to "bundle exec rake factory_girl:lint RAILS_ENV='test'" so that we don't need to include that gem for test environment as well.